### PR TITLE
added pallet

### LINKED ISSUE #351: Add bifrost farming pallet to the Pendulum runtime
## Context 

With the success of the generic AMM (Zenlink) on Amplitude, there is a business need to implement it on Pendulum. The bifrost farming pallet was already added and [configured](https://github.com/pendulum-chain/pendulum/blob/f5e92519105706925f7e9da6fae18079a3833017/runtime/foucoco/src/lib.rs#L1371-L1380) for the Foucoco and Amplitude runtime a while ago. 

## To-do
Add Bifrost's farming pallet and configure it the same way it is configured in the Amplitude runtime

## Acceptance Criteria

- Root user can create new pools for existing assets

### PR DESCRIPTION
**Pull request by gianfra-t**
@pendulum-chain/product: This PR adds changes to the node client code that require a **redeployment of the collator nodes** to take effect. This is only due to an addition of rpc call.

Closes #351.

### CODE REVIEW

File: runtime/pendulum/src/lib.rs [hunk 1]
```diff
@@ -908,6 +912,23 @@ impl dia_oracle::Config for Runtime {
 	type WeightInfo = dia_oracle::weights::DiaWeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const FarmingKeeperPalletId: PalletId = PalletId(*b"fo/fmkpr");
+	pub const FarmingRewardIssuerPalletId: PalletId = PalletId(*b"fo/fmrir");
```
Comment (ebma) [inline runtime/pendulum/src/lib.rs:917]: Nit: The `fo` indicates foucoco, on Amplitude we use `am` so let's use `pe` here.
```suggestion
	pub const FarmingKeeperPalletId: PalletId = PalletId(*b"pe/fmkpr");
	pub const FarmingRewardIssuerPalletId: PalletId = PalletId(*b"pe/fmrir");
```

**Review by ebma — COMMENTED** [review]
Looks good overall 👍 please change the title of the PR though.
Did you test that the RPC works? I remember that you said something didn't work about it @gianfra-t?

File: runtime/pendulum/src/lib.rs (hunk 1 above)
Comment (gianfra-t) [reply runtime/pendulum/src/lib.rs:917]: Oh I didn't notice. Committing.

**Comment by gianfra-t** [comment]
Yes you can call the RPC. It was probably a problem of how I was running the node. It did not appear in polkadot.js but again zenlink RPC was also not there. But calling it via curl worked!

**Review by ebma — APPROVED** [review]
Alright, thanks 👍  ready for merge once CI passed

**Comment by ebma** [comment]
@pendulum-chain/product did you see the note that this change requires a redeployment of the collator nodes to take effect?

**Pull request merged**